src/assets/scss/checkbox-component.scss

```diff
@@ -0,0 +1,12 @@
+.form-check-input {
+    background-color: $primary-light;
+    border: 1px solid $other-dark;
+    margin-bottom: 15px;
+}
+.form-check-input:checked {
+    background-color: $accent-light;
+    border-color: $accent-light;
+}
+.container-checkbox-component p{
```

Comment:
This class should have only css for checkbox

Reply:
It was not necessary i removed this class.